style/care.css

```
@@ -0,0 +1,69 @@
+
+.intro {
+    margin-top: 1em;
+    height: 275px;
+    display: flex;
+    width:100%;  
+}
+
+#summary {
+    width:55%;
+    background-color: #436105;
+    background-image: url(../assets/intro.jpg);
+    background-size: cover;
+
+
+}
```

```suggestion
    background-size: cover;
}
```

Remove unnecessary blank lines